test/lib/buildFactory.test.js

```diff
@@ -193,6 +212,17 @@ describe('Build Factory', () => {
             environment: { NODE_ENV: 'test', NODE_VERSION: '6' },
             image: 'node:6'
         }];
+        const permutationsWithAnnotaions = [{
```

Comment:
```suggestion
        const permutationsWithAnnotations = [{
```
(spelling typo with "annotations")